core/lib/l1_contract_interface/Cargo.toml

```diff
@@ -17,4 +17,15 @@ zksync_prover_interface = { path = "../prover_interface" }
 # Used to serialize proof data
 codegen = { git = "https://github.com/matter-labs/solidity_plonk_verifier.git", branch = "dev" }
 # Used to calculate commitment for vk from the old L1 verifier contract (backward comatibility needs)
-zkevm_test_harness = { git = "https://github.com/matter-labs/era-zkevm_test_harness.git", branch = "v1.3.3" }
+zkevm_test_harness_1_3_1 = { package = "zkevm_test_harness", git = "https://github.com/matter-labs/era-zkevm_test_harness.git", branch = "v1.3.3" }
```

Comment:
Just a tiny typo but for consistency should probably call it `zkevm_test_harness_1_3_3`? because of the branch name